fullbuild.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 set -ex
 swiftlint --strict
+
+# Make sure the version numbers on the podspec and SplunkRum.swift match
+echo "Check 3"
```

Comment:
`Checking that version numbers match` or similar would be a better log line than `Check 3`